### PR TITLE
cleans and simplifies installation process

### PR DESCRIPTION
**Pull request by Elorfin**

### CODE REVIEW

**Review by maieutiquer — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -30,8 +30,8 @@ file. For an installation from scratch, the commands would be:
 
     git clone -b 13.x http://github.com/claroline/Claroline
     cd Claroline
-    php bin/check
     php bin/configure
+    php bin/check
     composer install --no-dev --optimize-autoloader
 
     npm install
```
Comment (maieutiquer) [inline README.md:37]: Do we need `npm run webpack` after this command or after `composer build`?

Comment (Elorfin) [reply README.md:37]: Based on your comments in https://github.com/claroline/Claroline/pull/1508, I've removed the `composer build` command : 
- `buildBootstrap` cmd no longer exists.
- `clearCache` is only required for update and is now run by `pre-update-cmd`.

I've also added the `npm run webpack` in the doc.

Thank you for all the comments.

Comment (maieutiquer) [reply README.md:37]: Thanks, but `npm run webpack` seems broken, at least for new installations, it crashes after those lines:
```
<s> [webpack.Progress] 70% building 3340/3341 modules 1 active /var/www/html/claroline/node_modules/babel-loader/lib/index.js??ref--4!/var/www/html/claroline/vendor/claroline/distribution/main/app/Resources/modules/icon/svg/components/icon.jsx
<s> [webpack.Progress] 70% building 3341/3341 modules 0 active 
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! claroline@1.0.0 webpack: `webpack --config=vendor/claroline/distribution/webpack.config.prod.js --progress --bail`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the claroline@1.0.0 webpack script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2020-09-17T13_27_30_839Z-debug.log
```
Let me know if you need help reproducing, it's always reproducible for me, both locally and in Github Actions.

Comment (Elorfin) [reply README.md:37]: It is fixed by https://github.com/claroline/Claroline/pull/1510/commits/17e3ddebc3d8a22cf384b4d680fa0c9b8d58d31b

**Pull request merged**